WordPress/gutenberg
# Command for save, delete and duplicate post types

## What?
See https://github.com/WordPress/gutenberg/issues/50407

## Why?
Currently save, delete and duplicate operations on an editing post is not available.

## How?
Added command for saving, deleting and duplicating post  from command box.

## Testing Instructions

- Go to Posts in the WordPress admin.
- Open a new or existing post to enter the block editor.
- In the block editor, open the Command Palette (by pressing Cmd + K on Mac).
- Check for the save, delete, duplicate  post command.
- Select the command and confirm it works as expected.
- If the post is not already in draft, click Save as Draft or Preview to set it to a non-published state.
- Then again check for the save, delete, duplicate  post command and verify.


## Screenshots or screencast 

https://github.com/user-attachments/assets/6a2daaff-0dfd-4eac-9a29-773fa6c4d0c8
